### PR TITLE
ET-1279: AMP

### PR DESCRIPTION
**Pull request by anand-venkatraman**

### CODE REVIEW

File: ads/pulsepoint.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    tag(global, data);
+  }
+}
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+function tag(global, data) {
+  writeScript(global, 'https://tag.contextweb.com/getjs.aspx?action=VIEWAD&cwpid=' + data.pid + '&cwtagid=' + data.tagid + '&cwadformat=' + data.width + 'X' + data.height);
```
Comment (ProfessorEugene) [inline ads/pulsepoint.js:40]: maybe extract this out into a var up top or even build it via `../ads/_config.js`

File: ads/pulsepoint.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+function tag(global, data) {
+  writeScript(global, 'https://tag.contextweb.com/getjs.aspx?action=VIEWAD&cwpid=' + data.pid + '&cwtagid=' + data.tagid + '&cwadformat=' + data.width + 'X' + data.height); 
+}
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+function headerBidding(global, data) {
+  loadScript(global, 'https://ads.contextweb.com/ht.js', () => {
```
Comment (ProfessorEugene) [inline ads/pulsepoint.js:48]: [ditto](https://github.com/pulsepointinc/amphtml/pull/1/files#r68796898)

**Comment by ProfessorEugene** [comment]
Looks great!

File: ads/pulsepoint.js (hunk 1 above)
Comment (anand-venkatraman) [inline ads/pulsepoint.js:40]: `_config.js` has been setup in this repo, to host the prefetch urls and the hostnames to connect to. We wont be abel to move this there.

**Pull request merged**